app/javascript/react/assets/styles/colors.ts

```diff
@@ -1,17 +1,17 @@
 const cta = "#ffe651";
 const theme = "#00b2ef";
 
-const white = "#fff";
-const grey100 = "#f4f6f9";
+const white = "#Fff";
+const grey100 = "#F4f6f9";
 const grey200 = "#C0C9D1";
 const grey300 = "#7D858C";
 const grey400 = "#3E4449";
```

Comment:
It's minor. 
We need to decide if we use `grey` or `gray`. Me and Kasia changed to `gray` in our PRs, but I checked and both are correct. `Gray` is American and `grey` is British. 
I'd go with `gray` bc AirCasting is american project but let's decide together @franiakat 🥸